user/user.go

```diff
@@ -17,56 +16,61 @@ import (
 )
 
 var (
-	validate         = validator.New()
-	EMAIL_NOT_VALID  = errors.New("Email address is not valid")
-	HASHING_FAILED   = errors.New("Failed to hashed password")
-	CONFIRM_PASSWORD = errors.New("Password and confirm password must be the same, confirm and try again!")
+	errEmailNotValid   = errors.New("email address is not valid")
+	errHashingFailed   = errors.New("failed to hashed password")
 )
 
-// Method to hash password
+// Method to hash password.
 func GenerateHashPassword(password string) (string, error) {
-	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
+	cost := 14
+	bytes, err := bcrypt.GenerateFromPassword([]byte(password), cost)
+
 	return string(bytes), err
 }
 
-// An end point to create new users
+// An end point to create new users.
 func (uh *UserHandler) Create(response http.ResponseWriter, request *http.Request) {
 	response.Header().Add("content-type", "application/json")
-	user_collection := "users"
+
+	userCollection := UserCollectionName
```

Comment:
There is no need to create a new object in memory when we can make use of the existing constant.

Just pass the constant to where it is needed instead of passing it to a new object.